src/App.js

```diff
@@ -1,25 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import Main from './components/Main.js';
```

Comment:
HI there!

For `App.js` We can go ahead and import in Header, Footer and Main.